rune/libenclave/check.go

```diff
@@ -1,12 +1,19 @@
-package libenclave // import "github.com/opencontainers/runc/libenclave"
+package libenclave // import "github.com/inclavare-containers/rune/libenclave"
 
 import (
 	"fmt"
+	"os"
 	"strings"
 
+	"github.com/inclavare-containers/rune/libenclave/attestation/sgx"
+	"github.com/inclavare-containers/rune/libenclave/enclaveConfigs"
```

Comment:
The package enclaveConfigs should be libenclave/configs. In order to avoid conflicting with libcontainers/configs, set the export name as enclaveConfigs.